modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/AbstractKafkaToIgniteCdcStreamer.java

```diff
@@ -107,9 +107,14 @@ public AbstractKafkaToIgniteCdcStreamer(Properties kafkaProps, KafkaToIgniteCdcS
         appliers = new ArrayList<>(streamerCfg.getThreadCount());
         runners = new ArrayList<>(streamerCfg.getThreadCount());
 
-        if (!kafkaProps.containsKey(ConsumerConfig.GROUP_ID_CONFIG))
+        String grp = kafkaProps.getProperty(ConsumerConfig.GROUP_ID_CONFIG);
+
+        if (grp == null)
```

Comment:
Can we make this check with `A.ensure` like other parameters?

Reply:
Fixed